src/store/index.js

```diff
@@ -2,3 +2,4 @@ export { default as useMessageStore } from './messageStore';
 export { default as useUserStore } from './userStore';
 export { default as useToastStore } from './toastStore';
 export { default as useMemberStore } from './memberStore';
+export { default as useTotpStore } from './TotpmodalStore';
```

Comment:
can we change this store name to -> `totpModalStore` just for consistency

Reply:
sure